install/src/main/java/org/jetbrains/bsp/bazel/install/cli/CliOptionsProvider.kt

```diff
@@ -165,6 +165,13 @@ class CliOptionsProvider(private val args: Array<String>) {
             .desc("Use bloop as the BSP server rather than bazel-bsp.")
             .build()
         cliParserOptions.addOption(useBloopOption)
+
+        val bazelWorkspaceRootDirOption = Option.builder(BAZEL_WORKSPACE_ROOT_DIR_OPT)
+            .longOpt("bazel_workspace")
+            .hasArg()
+            .desc("Path to directory where workspace is located.")
```

Comment:
lets add what is the default - the `-d` and if `-d` is not specified then it's a pwd